### PR TITLE
gui: bump bicoind version to 27.1

### PR DESCRIPTION
**Pull request by pythcoiner**

### CODE REVIEW

File: gui/src/app/view/recovery.rs [hunk 1]
```diff
@@ -97,7 +97,7 @@ pub fn recovery<'a>(
                     Column::new()
                         .spacing(20)
                         .push(text(format!(
-                            "{} recovery paths will be available at the next block, select one:",
+                            "{} recovery paths will be available, select one:",
```
Comment (jp1ac4) [inline gui/src/app/view/recovery.rs:100]: Should this change be removed?

File: gui/src/bitcoind.rs [hunk 2]
```diff
@@ -22,22 +22,22 @@ use std::os::windows::process::CommandExt;
...
-pub const SHA256SUM: &str = "8a8e415763b7ffd5988153cf03967d812eca629016dd3b0ddf6da3ab6f4a3621";
+pub const SHA256SUM: &str = "06bf5d34aef01f21dac88708efa32d5e673c3ce1c8f17b1ca34b0e517ed8e43f";
 
 #[cfg(all(target_os = "linux", target_arch = "x86_64"))]
-pub const SHA256SUM: &str = "a5b7d206384a8100058d3f2e2f02123a8e49e83f523499e70e86e121a4897d5b";
+pub const SHA256SUM: &str = "c9840607d230d65f6938b81deaec0b98fe9cb14c3a41a5b13b2c05d044a48422";
 
 #[cfg(all(target_os = "windows", target_arch = "x86_64"))]
-pub const SHA256SUM: &str = "7bd0849e47472aeff99a0ea2c0cefd98f5be829e5a2d3b0168b5a54456cc638a";
+pub const SHA256SUM: &str = "e8ab7a8161c5a0846c81275809c9f90302874cc71b4378dee540f0b380a6d049";
```
Comment (jp1ac4) [inline gui/src/bitcoind.rs:40]: This should be for the win64.zip file.

File: gui/src/bitcoind.rs [hunk 3]
```diff
@@ -22,22 +22,22 @@ use std::os::windows::process::CommandExt;
...
 /// Current managed bitcoind version for new installations.
 pub const VERSION: &str = VERSIONS[0];
 
 #[cfg(all(target_os = "macos", target_arch = "x86_64"))]
-pub const SHA256SUM: &str = "acb50edd20692a9d023de12da573b64ca0fd9b4e9a2b88d1251020a3022b0f27";
+pub const SHA256SUM: &str = "25445080e2616d7c621f48f51e7a4294698d92016ef27ab88608cbe824f93364";
 
 #[cfg(all(target_os = "macos", target_arch = "aarch64"))]
-pub const SHA256SUM: &str = "8a8e415763b7ffd5988153cf03967d812eca629016dd3b0ddf6da3ab6f4a3621";
+pub const SHA256SUM: &str = "06bf5d34aef01f21dac88708efa32d5e673c3ce1c8f17b1ca34b0e517ed8e43f";
```
Comment (jp1ac4) [inline gui/src/bitcoind.rs:34]: The two macos values should be for the tar.gz files.

**Review by jp1ac4 — CHANGES_REQUESTED** [review]
Added comments for some required changes.

File: gui/src/bitcoind.rs [hunk 4]
```diff
@@ -22,22 +22,22 @@ use std::os::windows::process::CommandExt;
...
 /// Current and previous managed bitcoind versions, in order of descending version.
-pub const VERSIONS: [&str; 4] = ["26.1", "26.0", "25.1", "25.0"];
+pub const VERSIONS: [&str; 5] = ["27.1", "26.1", "26.0", "25.1", "25.0"];
 
 /// Current managed bitcoind version for new installations.
 pub const VERSION: &str = VERSIONS[0];
 
 #[cfg(all(target_os = "macos", target_arch = "x86_64"))]
-pub const SHA256SUM: &str = "acb50edd20692a9d023de12da573b64ca0fd9b4e9a2b88d1251020a3022b0f27";
+pub const SHA256SUM: &str = "25445080e2616d7c621f48f51e7a4294698d92016ef27ab88608cbe824f93364";
```
Comment (jp1ac4) [inline gui/src/bitcoind.rs:31]: This should also be for the tar.gz file.

Comment (pythcoiner) [reply gui/src/bitcoind.rs]: sorry, done

**Review by jp1ac4 — APPROVED** [review]
ACK 2514d0dedba61b6a9f546bedbe122c30614d7a52.

Tested on Linux with a new datadir and also an existing datadir where an older managed bitcoind version had already been installed for a different network.

**Pull request merged**